### ISSUE TITLE
Feature request: get display geometry

### ISSUE THREAD

**Issue by wael444** [naelstrof/slop#161]
mainly went here as this could be useful for x11grab in ffmpeg and maim.

**Comment by naelstrof** [comment]
Ah yeah I see what you mean, could be nice. You could already implement this by switching slop to pixel mode, then using xrandr output to determine which monitor you selected.

I wrote an example a while ago here: https://gist.github.com/naelstrof/544d76b0649512ee2458da4e6569278f 

Though I understand this script is a little intense and it would be nice if there was a more embedded feature.

**Comment by wael444** [comment]
yep, it is VERY intense, even as a shell programmer this is nearly unreadable for me. 

what does the script do exactly?

**Comment by naelstrof** [comment]
Yeah whoops. It essentially acts exactly like `maim -s`, except if you click on the background it shoots the monitor.

It could be far far simpler, just set slop to pixel mode and then use xrandr to parse the desktop geo and do a collision test.
